ftplugin/markdown.vim

```diff
@@ -819,7 +819,7 @@ function! s:MarkdownHighlightSources(force)
             else
                 let include = '@' . toupper(filetype)
             endif
-            let command = 'syntax region %s matchgroup=%s start="^\s*```\s*%s.*$" matchgroup=%s end="\s*```$" keepend contains=%s%s'
+            let command = 'syntax region %s matchgroup=%s start="^\s*```\s*%s.*$" matchgroup=%s end="^\s*```$" keepend contains=%s%s'
```

Comment:
Could you rebase this to the latest commit? #534 made some changes to this line.